DNM: Trying rattler multi-output

carterbox opened the pull request:
*DO NOT MERGE* Instead integrate these changes into the next release.

<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

carterbox committed:
DEV: Prototype multi-output recipe in new format

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found some lint.

Here's what I've got...


For **recipe/recipe.yaml**:

 * ❌ The top level meta key cache is unexpected
 * ❌ The recipe must have a `build/number` section.
 * ❌ There are 1 too many lines.  There should be one empty line at the end of the file.
 * ❌ This recipe is using a compiler, which now requires adding a build dependence on `${{ stdlib("c") }}` as well. Note that this rule applies to each output of the recipe using a compiler. For further details, please see https://github.com/conda-forge/conda-forge.github.io/issues/2102.

For **recipe/recipe.yaml**:

 * ℹ️ It looks like the '???' output doesn't have any tests.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/12638275656](https://github.com/conda-forge/conda-forge-webservices/actions/runs/12638275656). Examine the logs at this URL for more detail.</sub>

carterbox commented:
I need to rerender with `conda_build_tool: "rattler-build"`, but this recipe requires the `--experimental` flag for rattler.

carterbox commented:
The inability to use experimental features with rattler-build has already been reported.

https://github.com/conda-forge/conda-smithy/issues/2057

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I was trying to look for recipes to lint for you, but it appears we have a merge conflict. Please try to merge or rebase with the base branch to resolve this conflict.

Please ping the 'conda-forge/core' team (using the `@` notation in a comment) if you believe this is a bug.

carterbox commented:
```
rattler-build build -m .ci_support/linux_64_.yaml -r recipe/recipe.yaml --experimental
```

carterbox committed:
MNT: Re-rendered with conda-smithy 3.61.1 and conda-forge-pinning 2026.04.21.11.53.26

Other tools:
- conda-build 26.3.0
- rattler-build 0.63.0
- rattler-build-conda-compat 1.4.13

carterbox committed:
STY: Remove extra line

carterbox committed:
BLD: Fixup build requirements

carterbox commented on recipe/recipe.yaml:
```diff
@@ -0,0 +1,204 @@
+        # If this output accepts run_exports from dav1d-split, then I would expect to not
+        # need to specify the build dependencies again?
+        - ${{ compiler('c') }}
+        - ${{ stdlib('c') }}
```
@wolfv Is it expected to need to supply these requirements here in order for the link checker to identify `libc.so.6` and `libpthread.so.0` as system dependencies? Even with `inherit/run_exports: true`?

wolfv replied:
Agreed, that seems like there is still a bug with the link checking from inherited dependencies. We'll have to look into it. I'll open an issue in rattler-build.

carterbox commented:
I guess we're dropping ppc64le. rattler-build is many versions behind on that platform.

carterbox committed:
Remove linux_ppc64le provider configuration

Remove linux_ppc64le provider from conda-forge.yml

carterbox commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/recipe.yaml**:

 * ℹ️ It looks like the '???' output doesn't have any tests.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/24842338117](https://github.com/conda-forge/conda-forge-webservices/actions/runs/24842338117). Examine the logs at this URL for more detail.</sub>

Commit:
MNT: Re-rendered with conda-smithy 3.61.1 and conda-forge-pinning 2026.04.23.11.42.2
